src/Gren/Kernel/Bytes.js

```diff
@@ -129,24 +129,24 @@ var _Bytes_write_string = F3(function(mb, offset, string)
 var _Bytes_decode = F2(function(decoder, bytes)
 {
 	try {
-		return __Maybe_Just(A2(decoder, bytes, 0).b);
+		return __Maybe_Just(A2(decoder, bytes, 0).value);
```

Comment:
This should also be `__$value`, shouldn't it?

Try testing whatever test app you have after compiling with `--optimize`